### ISSUE TITLE
Error: npm install express

### ISSUE THREAD

**Issue by ko25july** [sandboxie-plus/Sandboxie#2878]
### Describe what you noticed and did

I'm develope web application with node js
1. Install Node JS v18.16.0
2. Open windows 11 command prompt
3. md test
4. cd test
5. npm init -y
6. npm install express
7. Error show that
8. `npm ERR! code EISDIR
npm ERR! syscall link`
`npm ERR! path C:\Users\KO\AppData\Local\npm-cache\_cacache\tmp\a1c4bd37`
`npm ERR! dest C:\Users\KO\AppData\Local\npm-cache\_cacache\content-v2\sha512\72\81\99c0ad3eb6c9ffe36109dcdd3daeb8fc647d9e23d5170c7dfc17ac6eb2d73fdc2fa9661d19a4411fb859e96ee2add67b09668635e762c7dbd479b6d51eff`
`npm ERR! errno EISDIR`
`npm ERR! Invalid response body while trying to fetch https://registry.npmjs.org/express: EISDIR: illegal operation on a directory, link 'C:\Users\KO\AppData\Local\npm-cache\_cacache\tmp\a1c4bd37' -> 'C:\Users\KO\AppData\Local\npm-cache\_cacache\content-v2\sha512\72\81\99c0ad3eb6c9ffe36109dcdd3daeb8fc647d9e23d5170c7dfc17ac6eb2d73fdc2fa9661d19a4411fb859e96ee2add67b09668635e762c7dbd479b6d51eff'`
`npm ERR! A complete log of this run can be found in:`
`npm ERR!     C:\Users\KO\AppData\Local\npm-cache\_logs\2023-05-01T13_30_18_008Z-debug-0.log`

### How often did you encounter it so far?

Every time that use command line of `npm install express`

### Affected program

Node JS v18.16.0 and NPM v9.5.1

### Download link

https://nodejs.org/dist/v18.16.0/node-v18.16.0-x64.msi

### Where is the program located?

The program is installed only inside a sandbox (NOT in the real system anyway).

### Expected behavior

`C:\>md Temp

C:\>cd Temp

C:\Temp>md test

C:\Temp>cd test

C:\Temp\test>npm init -y
Wrote to C:\Temp\test\package.json:

{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}

C:\Temp\test>npm install express

added 57 packages, and audited 58 packages in 6s

7 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice
npm notice New minor version of npm available! 9.5.0 -> 9.6.5
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.6.5
npm notice Run npm install -g npm@9.6.5 to update!
npm notice`

### What is your Windows edition and version?

Windows 11 Pro 21H2 64bit

### In which Windows account you have this problem?

I use the built-in Administrator account.

### Please mention any installed security software

Microsoft Defender Antivirus

### What version of Sandboxie are you running?

Sandboxie Plus x64 v1.9.2

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### Sandboxie.ini configuration

```shell
#
# Sandboxie configuration file
#

[GlobalSettings]
NetworkEnableWFP=y
DefaultBox=DefaultBox
FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\%SANDBOX%
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
Template=WindowsLive
Template=OfficeLicensing

[UserSettings_028E00DB]
SbieCtrl_AutoStartAgent=SandMan.exe
SbieCtrl_EnableAutoStart=y
SbieCtrl_ShouldDeleteNotify=y

[DefaultBox]
Enabled=y
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=10
UseFileDeleteV2=y
UseRegDeleteV2=y
```

**Comment by czenzel** [comment]
I also can confirm this is happening with node when restoring from a packages file with Sandboxie Plus.

**Comment by ko25july** [comment]
@czenzel thank you very much 👍👍👍

**Comment by czenzel** [comment]
@ko25july A workaround for now would be to restore the packages inside another virtual machine or folder and then copy them into the Sandbox Contents. That is the way I got around on my setup.

**Comment by ko25july** [comment]
@czenzel i have tried able to work Although it's a bit complicated. And thanks very much for your very good advice.

**Comment by offhub** [comment]
It gives an error because it cannot create a symlink in the sandbox.

**Comment by ko25july** [comment]
@offhub Thank you very much for your help and for knowing the cause of this error.

**Comment by offhub** [comment]
This issue seems to have been fixed in NodeJS v18.17. If you can verify this, we can close this issue.

**Comment by ko25july** [comment]
@offhub This issue have been fixed in NodeJS v18.17.
It work perfect. Thank you very much.
